_includes/tutorials/anomaly-detection/ksql/code/docker-compose.yml

```diff
@@ -43,7 +43,7 @@ services:
       SCHEMA_REGISTRY_KAFKASTORE_BOOTSTRAP_SERVERS: 'broker:9092'
 
   ksqldb-server:
-    image: confluentinc/ksqldb-server:0.11.0
+    image: confluentinc/ksqldb-server:0.13.0
```

Comment:
Why 0.13.0 here?

Reply:
There appears to be a join timing issue with ksql-test-runner above 0.13.0. I propose we address this in a follow up, rather than block the overall update.

Reply:
@colinhicks -- if it is desired to leave this one test case at `0.13.0` and push the PR through with it, can you please add a comment into the docker-compose.yml file explaining the issue with a link to ksqldB GitHub issue?  Just so we remember in the future why this one is different